ui_src/src/components/customSelect/index.js

```diff
@@ -0,0 +1,130 @@
+
+    return (
+        <div className="select-schema-container">
+            <Select
+                className="select"
+                placeholder={placeholder}
+                value={displayValue}
+                bordered={false}
+                suffixIcon={<ArrowDropDownRounded className="drop-down-icon" />}
+                onChange={handleChange}
+                placement="bottomRight"
+                popupClassName="select-schema-options"
+                notFoundContent={
+                    type === 'schema' ? (
+                        <div className="no-schema-to-display">
+                            <img src={placeholderSchema} width="50" height="50" alt="placeholderSchema" />
+                            <p className="title">No schemas yet</p>
+                            <p className="sub-title">Get started by creating your first schema</p>
+                            <Button
+                                className="modal-btn"
+                                width="120px"
+                                height="34px"
+                                placeholder="Create schema"
+                                colorType="white"
+                                radiusType="circle"
+                                backgroundColorType="purple"
+                                fontSize="12px"
+                                fontFamily="InterSemiBold"
+                                aria-controls="usecse-menu"
+                                aria-haspopup="true"
+                                onClick={() => createNew()}
+                            />
+                        </div>
+                    ) : type === 'user' ? (
+                        <div className="no-schema-to-display">
+                            <div className="placeholder-background">
+                                <img src={usersIconActive} width={40} height={40} alt="placeholderSchema" />
+                            </div>
+                            <p className="title">No users yet</p>
+                            <p className="sub-title">Get started by creating your first user</p>
+                            <Button
+                                className="modal-btn"
+                                width="120px"
+                                height="34px"
+                                placeholder="Create user"
+                                colorType="white"
+                                radiusType="circle"
+                                backgroundColorType="purple"
+                                fontSize="12px"
+                                fontFamily="InterSemiBold"
+                                aria-controls="usecse-menu"
+                                aria-haspopup="true"
+                                onClick={handleCreateNew}
+                            />
+                        </div>
+                    ) : null
+                }
+            >
+                {options?.map((schema) => {
```

Comment:
The same here.
Also, change the name to something more generic.

Reply:
It was unclear when we called the item "schema" because we used this component for more types.
Please use a more generic name like "option."